src/content/ui-api/management-api/authentication/index.md

```diff
@@ -123,11 +123,87 @@ Let's go into details for the most relevant properties (also called "claims") of
 
-Each Giant Swarm installation provides a service account named `automation` in the `default` namespace.
+### Remote access to the Management Cluster {#remote-mc}
 
-**Note:** This service account comes with a quite powerful set permissions. **We strongly recommend to create a specific service account for each application**, binding it to specific roles granting only the required permissions in the required namespaces.
+Each Giant Swarm installation provides a service account named `automation` in the `default` namespace that may be used for creating a self-contained `kubeconfig` file. **Note however**, this account comes with a quite powerful set of permissions, thus **we strongly recommend to create a specific service account for each application**, binding it to specific roles granting only the required permissions in the required namespaces.
+
+Regardless of service account you decide to use, and assuming you are currently logged in to the right installation, you can utilize the below instruction to create a config file.
+
+1. Export the Service Account name:
+
+```nohighlight
+SANAME=automation
+```
+
+2. Find Kubernetes Secret associated with the service account:
+
+```nohighlight
+SECRET=$(kubectl get sa $SANAME -o jsonpath='{.secrets[0].name}')
+```
+
+3. Find the service account's token:
+
+```nohighlight
+TOKEN=$(kubectl get secret $SECRET -o jsonpath='{.data.token}' | base64 --decode)
+```
+
+4. Find Kubernetes CA certificate:
+
+```nohighlight
+CACERT=$(kubectl get secret $SECRET -o jsonpath='{.data.ca\.crt}')
+```
+
+5. Grab installation's API endpoint:
+
+```nohighlight
+APIURL=$(kubectl config view --minify -o jsonpath='{.clusters[0].cluster.server}')
+```
+
+6. Grab installation's name:
+
+```nohighlight
+CODENAME=$(kubectl config view --minify -o jsonpath='{.clusters[0].name}')
+```
+
+7. Generate kubectl configuration:
+
+```nohighlight
+cat <<EOF > kubeconfig
+apiVersion: v1
+kind: Config
+clusters:
+  - name: $CODENAME
+    cluster:
+      certificate-authority-data: $CACERT
+      server: $APIURL
+users:
+  - name: $SANAME
+    user:
+      token: $TOKEN
+current-context: $CODENAME
+contexts:
+- context:
+    cluster: $CODENAME
+    user: $SANAME
+  name: $CODENAME
+EOF
+```
+
+8. Test newly created file:
+
+```nohighlight
+kubectl --kubeconfig kubeconfig cluster-info
+```
+
+### Remote access to the Workload Cluster {#remote-wc}
+
+Workload Clusters relies on the [TokenRequest API](https://kubernetes.io/docs/reference/kubernetes-api/authentication-resources/token-request-v1/#create-create-token-of-a-serviceaccount) for authentication, so instruction from the previous paragraph will not work against them. We strongly encourage you to use the `kubectl-gs` plugin and [client certificate login with a self-contained kubeconfig](https://docs.giantswarm.io/ui-api/kubectl-gs/login/#wc-client-cert-automation) instead.
```

Comment:
```suggestion
Workload clusters rely on the [TokenRequest API](https://kubernetes.io/docs/reference/kubernetes-api/authentication-resources/token-request-v1/#create-create-token-of-a-serviceaccount) for authentication, so the steps from the previous paragraph will not work against them. We strongly encourage you to use the `kubectl-gs` plugin and [client certificate login with a self-contained kubeconfig](https://docs.giantswarm.io/ui-api/kubectl-gs/login/#wc-client-cert-automation) instead.
```

Which part is internal? That isn't clear for me but as long as we're linking to public docs I think its fine.

Reply:
Probably none, I mean I haven't found any reference in our docs that we use TokenRequest API and do not inject SAs token directly into containers on WCs, so I wondered if that's something I can share in public docs. But then I guess users on WCs have admin privileges, so they can check configuration of the API server themselves and grab this info?

Reply:
Hmm I'm not sure on our usage of the TokenRequest API. I think Rainbow would know better.

In general though we like to link to the upstream Kubernetes docs when appropriate. So good to include it. 👍

Reply:
@giantswarm/team-rainbow sorry for bothering you, I could really use your opinion here 🙏

Reply:
I don't know enough here. (Hoping for the rest of the team)